### PR TITLE
Fixes multiple UI issues.

### PR DESCRIPTION
**Pull request by percy-g2**

### CODE REVIEW

**Comment by percy-g2** [comment]
Have UI testing on android, ios, mac, linux and windows.

File: shared/src/commonMain/kotlin/com/myprojectname/app/ui/components/PetCardSmall.kt [hunk 1]
```diff
@@ -34,19 +35,18 @@ import com.myprojectname.app.ui.theme.Typography
 import io.kamel.image.KamelImage
 import io.kamel.image.asyncPainterResource
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun PetCardSmall(
     item: PetModel,
-    width: Dp,
```
Comment (ferranpons) [inline shared/src/commonMain/kotlin/com/myprojectname/app/ui/components/PetCardSmall.kt:40]: Why you removed the option of setting a custom size for the Pet Card? This that this component is used in 2 places that need a different size.

File: shared/src/commonMain/kotlin/com/myprojectname/app/ui/components/PetCardBig.kt [hunk 2]
```diff
@@ -33,20 +33,18 @@ import com.myprojectname.app.ui.theme.Typography
...
 ) {
-    val imageHeight = 120.dp
-    val roundedCornerSize = 10.dp
-
     Card(
         onClick = { onClick() },
         modifier = Modifier
             .padding(start = 16.dp, end = 16.dp, bottom = 16.dp)
             .fillMaxWidth(),
-        shape = RoundedCornerShape(roundedCornerSize),
```
Comment (ferranpons) [inline shared/src/commonMain/kotlin/com/myprojectname/app/ui/components/PetCardBig.kt:49]: This was a constant? Why hard code the values?

File: shared/src/commonMain/kotlin/com/myprojectname/app/feature/profile/ProfileDetailScreen.kt [hunk 3]
```diff
@@ -209,13 +207,12 @@ class ProfileDetailScreen(val userId: Int) : Screen {
                     )
 
                     if (state.pets.isNotEmpty()) {
-                        val itemSize: Dp = (maxWidth / 2) - 8.dp
```
Comment (ferranpons) [inline shared/src/commonMain/kotlin/com/myprojectname/app/feature/profile/ProfileDetailScreen.kt:212]: If you remove this the view will not adapt its size depending the screen size.

File: shared/src/commonMain/kotlin/com/myprojectname/app/ui/components/PetCardBig.kt [hunk 4]
```diff
@@ -33,20 +33,18 @@ import com.myprojectname.app.ui.theme.Typography
 import io.kamel.image.KamelImage
 import io.kamel.image.asyncPainterResource
 
+@OptIn(ExperimentalMaterial3Api::class)
```
Comment (ferranpons) [inline shared/src/commonMain/kotlin/com/myprojectname/app/ui/components/PetCardBig.kt:36]: The opt in is in file level (look at the top of the file) no need to add it to the method. If you have noticed that the IDE is complaining is because there is a problem with the IDE linter but not in the code. Hope Jetbrains solve it any time soon.

File: shared/src/commonMain/kotlin/com/myprojectname/app/ui/screens/HomeTabScreen.kt [hunk 5]
```diff
@@ -278,29 +275,28 @@ class HomeTabScreen : Screen {
...
             text = localization.homePetsNearYou,
             style = Typography.get().titleMedium,
             modifier = Modifier.padding(start = 16.dp, end = 16.dp, top = 16.dp)
         )
         if (state.nearMeAds.isNotEmpty()) {
-            val itemSize: Dp = (maxWidth / 2) - 8.dp
-            FlowRow(
-                horizontalArrangement = Arrangement.SpaceEvenly,
-                maxItemsInEachRow = 2
+            LazyRow(
```
Comment (ferranpons) [inline shared/src/commonMain/kotlin/com/myprojectname/app/ui/screens/HomeTabScreen.kt:285]: Why this change?

File: shared/src/commonMain/kotlin/com/myprojectname/app/ui/screens/PetDetailScreen.kt [hunk 6]
```diff
@@ -73,7 +76,13 @@ fun PetDetailView(petModel: PetModel, onClose: () -> Unit) {
                         )
                     }
                 },
-                title = {},
+                title = {
+                    Text(
+                        text = "Back",
```
Comment (ferranpons) [inline shared/src/commonMain/kotlin/com/myprojectname/app/ui/screens/PetDetailScreen.kt:81]: The project has i18n, you have to use the correct localized string.

**Review by ferranpons — CHANGES_REQUESTED** [review]
First of all, many thanks for your contribution. I appreciate the time you spent with the project.

You have to know that I was not currently accepting contributions (it's in the README), even though, I just reviewed your PR.

I'm sorry but this PR does not meet the requirements to be merged for some reasons.

- You need to provide more information when submitting a PR, I need to understand why you made those changes. Next time add some screenshots and a description of the changes.
-  You need to follow the guidelines of the project, and/or if not described, follow the same rules the code is using.
- You need to understand in which state is the project right now, it is not finished or does not meet the features and code status I think it would make it "stable".

I hope it does not sound "rude", it's not my intention. I really thank you for taking the time to try the code and make some changes but you need to understand that it will need to meet some requirements before a PR could be merged.

**Comment by ferranpons** [comment]
@percy-g2 I'm closing this PR since there are a lot of changes since you submitted it. Once again, many thanks for taking the time for making those changes.
